pkgs/tools/X11/opentabletdriver/default.nix

```diff
@@ -76,6 +76,8 @@ buildDotnetModule rec {
     "OpenTabletDriver.Tests.ConfigurationTest.Configurations_DeviceIdentifier_IsNotConflicting"
     # Depends on processor load
     "OpenTabletDriver.Tests.TimerTests.TimerAccuracy"
+    # Can't find Configurations directory
+    "OpenTabletDriver.Tests.ConfigurationTest.Configurations_Verify_Configs_With_Schema"
```

Comment:
This test fails, I assume because it tries to go out from the OpenTabletDriver.Tests directory to the OpenTabletDriver.Configuration directory which doesn't work as expected with nix: https://github.com/OpenTabletDriver/OpenTabletDriver/blob/v0.6.4.0/OpenTabletDriver.Tests/ConfigurationTest.cs#L236
<details><summary>Test stack trace:</summary>
<pre>
  Failed OpenTabletDriver.Tests.ConfigurationTest.Configurations_Verify_Configs_With_Schema [91 ms]
  Error Message:
   System.TypeInitializationException : The type initializer for 'OpenTabletDriver.Tests.ConfigurationTest' threw an exception.
---- System.IO.DirectoryNotFoundException : Could not find a part of the path '/build/source/OpenTabletDriver.Tests/OpenTabletDriver.Configurations/Configurations'.
  Stack Trace:
     at OpenTabletDriver.Tests.ConfigurationTest.Configurations_Verify_Configs_With_Schema() in /build/source/OpenTabletDriver.Tests/ConfigurationTest.cs:line 250
----- Inner Stack Trace -----
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String directory, Boolean isNormalized, EnumerationOptions options)
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options, Boolean isNormalized)
   at System.IO.Enumeration.FileSystemEnumerableFactory.UserFiles(String directory, String expression, EnumerationOptions options)
   at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at System.IO.Directory.EnumerateFiles(String path, String searchPattern, SearchOption searchOption)
   at OpenTabletDriver.Tests.ConfigurationTest..cctor() in /build/source/OpenTabletDriver.Tests/ConfigurationTest.cs:line 238
</pre></details>
@thiagokokada  @InfinityGhost should this test be disabled or you know how to fix it?

Reply:
I think disabling the test is fine, the NixOS tests are more important anyway.